package Ulab;

public class FinalException {
	
	public static void main(String[] args) {
		try {
			throw new ArrayIndexOutOfBoundsException();
		}
		catch(ArrayIndexOutOfBoundsException e) {
			System.out.println("Array index out of bound exception");
			try {
			throw new NullPointerException();
		}
			catch(NullPointerException f) {
				
				System.out.println("Null pointer exception");
			}
		
		}
		
		finally {
			try {
			throw new X();
		}
		catch(X n) {
			System.out.println("Arithmetic Expression");
		}
		

}
	
}
}

//

package Ulab;

  class X extends ArithmeticException {
	
	}

